src/add-to-registry.js

```diff
@@ -36,13 +36,16 @@ const { isInRegistry, addToRegistry, getFromRegistry, updateInRegistry, TEMPLATE
             "categories": npmPackageMetadata.categories,
             "adobeRecommended": adobeRecommended,
             "keywords": npmPackageMetadata.keywords,
-            "status": TEMPLATE_STATUS_APPROVED,
             "links": {
                 "npm": npmUrl,
                 "github": gitHubUrl
             }
         }
-
+        if(adobeRecommended) {
```

Comment:
@AmyJZhao , we shouldn't use `adobeRecommended` for deciding if a template should be approved or not. Logic for setting up `adobeRecommended` might be changed, and that shouldn't affect the approval process. We could use a previous logic with checking a number of _stars_, a 3rd-party repo might have many stars, but that doesn't mean that we should approve that template automatically.